test/unit/sql/json.test.js

```diff
@@ -10,44 +10,49 @@ const Support = require(__dirname + '/../support')
   , sql = current.dialect.QueryGenerator;
 
 // Notice: [] will be replaced by dialect specific tick/quote character when there is not dialect specific expectation but only a default expectation
-
 if (current.dialect.supports.JSON) {
+  const type = DataTypes[current.dialect.name].JSON ?
```

Comment:
Why this need special treatment for MySQL, it seems to work with Postgres and SQLite ?